formatters/simplecov/simplecov_test.go

```diff
@@ -9,6 +9,33 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func Test_ParseLegacy(t *testing.T) {
+	ogb := env.GitBlob
+	defer func() {
+		env.GitBlob = ogb
+	}()
+	env.GitBlob = func(s string, c *object.Commit) (string, error) {
+		return s, nil
+	}
+
+	r := require.New(t)
+
+	f := Formatter{
+		Path: "./simplecov-example-legacy-resultset.json",
+	}
+	rep, err := f.Format()
+	r.NoError(err)
+
+	r.Len(rep.SourceFiles, 7)
```

Comment:
The single char vars here are making my eye itchy  🤨 
Can we make the names a bit more descriptive? Like, 4 lines down and I don't recall what `r` is anymore 😬

Reply:
hahah, yeah I know sorry, I kinda follow the pattern that was already in place, but I have just update all the tests to use better namings.